examples/asr/conf/transformer_dec/speech_translation_transf.yaml

```diff
@@ -0,0 +1,235 @@
+# It contains the default values for training a Conformer-CTC ASR model, large size (~120M) with CTC loss and sub-word encoding.
```

Comment:
Hmm wondering if it should be conf/speech_translation/fastconformer/speech_translation_transformer.yaml ?